### PR TITLE
fix dashboard horizontal scrollbar

### PR DESCRIPTION
**Pull request by hbrwang**
fix dashboard horizontal scrollbar

### CODE REVIEW

File: web/frps/src/App.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
             </el-row>
         </header>
         <section>
-            <el-row :gutter="20">
```
Comment (fatedier) [inline web/frps/src/App.vue:9]: What's the problem of this attribute?

Can you paste screenshots to describe what's the difference?

Comment (fatedier) [reply web/frps/src/App.vue:9]: I have not seen what's the problem.

Comment (hbrwang) [reply web/frps/src/App.vue:9]: 这个 `:gutter="20"` 会给 `el-row` 添加

``` 
 style="margin-left: -10px; margin-right: -10px;"
```

导致网页始终出现横向滚动条。手机浏览器则表现为 右侧多出空白竖条，强迫症表示很难受。

**Review by fatedier — APPROVED** [review]

**Pull request merged**